src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/HelpListener.java

```diff
@@ -30,13 +34,35 @@ public HelpListener(MainJFrame window) {
 
     @Override
     public void actionPerformed(ActionEvent e) {
-
-        //TODO: make user manual work
-        // create a new frame with an embedded user manual
-        JFrame helpFrame = new JFrame("User Manual");
-        helpFrame.setSize(500, 500);
-        JLabel label = new JLabel("Will be implemented soon!", JLabel.CENTER);
-        helpFrame.add(label);
-        helpFrame.setVisible(true);
+    	open(createURI(USER_MANUAL_LINK));
+    	
+    }
+    /**
+	 * Opens the URI in the URI object using the desktop's default browser
+	 * @param URI uri
+	 */
+	public static void open(URI uri){
```

Comment:
[DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is violated here, perhaps open/createURI should be in some utility class?